libflux/src/flux/semantic/builtins.rs

```diff
@@ -301,6 +301,178 @@ pub fn builtins() -> Builtins<'static> {
+                "drop" => Node::Builtin(r#"
+                    forall [t0, t1, t2] where t0: Row, t1: Row, t2: Row (
+                        <-tables: [t0],
+                        ?fn: (column: string) -> bool,
+                        ?columns: t1
+                    ) -> [t2]
+                "#),
+                "duplicate" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        column: string,
+                        as: string
+                    ) -> [t1]
+                "#),
+                "duration" => Node::Builtin("forall [t0] (v: t0) -> duration"),
+                "elapsed" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?unit: duration,
+                        ?timeColumn: string,
+                        ?columnName: string
+                    ) -> [t1]
+                "#),
+                "exponentialMovingAverage" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        n: int
+                    ) -> [t1]
+                "#),
+                "false" => Node::Builtin("forall [] () -> bool"),
+                "fill" => Node::Builtin(r#"
+                    forall [t0, t1, t2] where t0: Row, t2: Row (
+                        <-tables: [t0],
+                        ?column: string,
+                        value: [t1],
+                        usePrevious: bool
+                    ) -> [t2]
+                "#),
+                "filter" => Node::Builtin(r#"
+                    forall [t0, t1, t2] where t0: Row, t2: Row (
+                        <-tables: [t0],
+                        fn: (r: t1) -> bool
+                    ) -> [t2]
+                "#),
+                "first" => Node::Builtin(r#"
+                    forall [t0] where t0: Row (
+                        <-tables: [t0],
+                        ?column: string
+                    ) -> [t0]
+                "#),
+                "float" => Node::Builtin("forall [t0] (v: t0) -> float"),
+                "getColumn" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        table: [t0],
+                        column: string
+                    ) -> [t1]
+                "#),
+                "getRecord" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        table: [t0],
+                        idx: int
+                    ) -> [t1]
+                "#),
+                "group" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?mode: string,
+                        ?columns: [string]
+                    ) -> [t1]
+                "#),
```

Comment:
Similarly for `group`, because it doesn't actually modify the input type, its type signature only needs one generic parameter:

    forall [t0] where t0: Row (<-tables: [t0], ?mode: string, ?columns: [string]) -> [t0]